### DISCUSSION TITLE
PocketBase in Vuejs

### DISCUSSION THREAD

**Discussion by matii3** [pocketbase/pocketbase#804] [Q&A]
Hi,
How can i fix this problem?

Module not found: Error: Can't resolve 'pocketbase' in 'C:\Users\matie\project35\src\components\Groups'
This is where the vue file is

**Comment by ganigeorgiev** [comment]
Isn't this the same as in https://github.com/pocketbase/pocketbase/discussions/767?

**Reply by ganigeorgiev** [reply]
Additionally, make sure that the PocketBase executable is not placed in the same directoy where you component is since sometimes, depending how your bundler loader is configured, it may try to load the binary file instead the js `node_modules/pocketbase` dependency as noted by another user in https://github.com/pocketbase/js-sdk/issues/13#issuecomment-1210653718.
